src/webviews/components/issue/JiraIssuePage.tsx

```diff
@@ -72,6 +72,17 @@ export default class JiraIssuePage extends AbstractIssueEditorPage<Emit, Accept,
         this.state = emptyState;
     }
 
+    // TODO: proper error handling in webviews :'(
+    // This is a temporary workaround to hopefully troubleshoot
+    // https://github.com/atlassian/atlascode/issues/46
+    getInputMarkup(field: FieldUI, editmode?: boolean, context?: String) {
```

Comment:
Refactor suggestion.

Goal: Reduce copy-paste error on the callers

```
getInputMarkup(fields: /*typeOf(this.state.fields)*/, fieldKey?: String, editmode?: boolean) {
     const field = fields[fieldKey];
     if (!field) {
          console.error(`Field error - no field when trying to render ${fieldKey}`);
          return null;
      }
      return super.getInputMarkup(field, editmode);
}
```

Reply:
Also, it looks like the `editmode` param is always true.. Maybe we can drop from the signature here?

Reply:
Unfortunately, we have limited flexibility with the signature here since this is an override from base class xD
I went this route to minimize changes made to code.

Added an explicit `override` for clarity though - and replaced `console.error` with `console.warn`:

![image](https://github.com/user-attachments/assets/3f6babd9-89f8-4bcd-8f90-7119e3bddac5)